CONTRIBUTING.md

```diff
@@ -18,12 +24,121 @@ Possible ways to contribute:
+* Ian Crosby, Container Solutions (ian.crosby@container-solutions.com)
+* Jeyappragash JJ, Independent (pragashjj@gmail.com)
+* Jinming Yue, Caicloud (yuejinming@caicloud.io)
+* Joe Beda, Heptio (joe@heptio.com)
+* John Hillegass, Capital One (john.hillegass@capitalone.com)
+* Jonghyuk Jong Choi, NCSoft (jongchoi@ncsoft.com)
+* Josef Adersberger, QAware (josef.adersberger@qaware.de)
+* Joseph Jacks, Independent	(jacks.joe@gmail.com)
+* Josh Bernstein, Dell (Joshua.Bernstein@dell.com)
+* Justin Cappos, NYU (jcappos@nyu.edu)
+* Justin Cormack, Docker (justin.cormack@docker.com)
+* Jun Du, Huawei (dujun5@huawei.com)
+* Kai Chen, Alauda (kaichen@alauda.io)
+* Kiran Mova, MayaData (kiran.mova@mayadata.io)
+* Körbächer, Max, Storm Reply (m.koerbaecher@reply.de)
+* Lachlan	Evenson, Microsoft (lachlan.evenson@microsoft.com)
+* Lee Calcote, SolarWinds (leecalcote@gmail.com)
+* Lei	Zhang, HyperHQ (harryzhang@zju.edu.cn)
+* Louis Fourie, Huawei (louis.fourie@huawei.com)
+* Mark Peek, VMware	(markpeek@vmware.com)
+* Matt Farina, Samsung SDS (matt@mattfarina.com)
+* Matthew Fornaciari, Gremlin (forni@gremlin.com)
+* Matt Young, EverQuote (myoung@everquote.com)
+* Nick Chase, Mirantis	(nchase@mirantis.com)
+* Nimesh Agarwal, Dunzo (nimesh.mittal@gmail.com)
+* Pengfei Ni, Microsoft (peni@microsoft.com)
+* Philip Lombardi, Datawire.io (plombardi@datawire.io)
+* Piyush Sharrma, Accurics (piyush@accurics.com)
+* Praveen Singh, Dunzo (singhpraveen2010@gmail.com)
+* Quinton Hoole, Huawei (quinton.hoole@huawei.com)
+* Randy	Abernethy, RX-M LLC (randy.abernethy@rx-m.com)
+* Ricardo Aravena, Rakuten (raravena80@gmail.com)
+* Rick Spencer, Bitnami	(rick@bitnamni.com)
+* Sarah Allen, Independent (sarah@ultrasaurus.com)
+* Shida Qiu, BoCloud (shidaqiu2018@gmail.com)
+* Siddharth Bhadri, Infoblox (sbhadri@infoblox.com)
+* Steve Dake, IBM (sdake@ibm.com)
+* Swamy D K V, Cisco (swamydkv@gmail.com)
+* Tammy Butow, Gremlin (tammy@gremlin.com)
+* Timothy Chen, Hyperpilot (tim@hyperpilot.io)
+* Vasu Chandrasekhara, SAP SE (vasu.chandrasekhara@sap.com)
+* Xiang Li, Alibaba (x.li@alibaba.com)
+* Xu Wang, Hyper (xu@hyper.sh)
+* Yaron Haviv, iguazio (yaronh@iguaz.io)
+* Yong Tang, Infoblox (ytang@infoblox.com)
+* Yuri Shkuro, Uber	(ys@uber.com)
+* Zefeng (Kevin) Wang, Huawei (wangzefeng@huawei.com)
+* Zou Nengren, CMCC (zounengren@cmss.chinamobile.com)
+* Jia Xuan, CMCC (jiaxuan@chinamobile.com)
+* Zhipeng Huang, Huawei (huangzhipeng@huawei.com)
 
-This is not only about individual contribution.  It is also about rallying help from your employer, e.g., if you work for a CNCF Member company. Given the [breadth](https://landscape.cncf.io/) of projects represented by cloud native, it is impossible for anyone to be an expert in all technologies that we’re evaluating. We’re particularly interested in Contributors that can act as a focal point for tapping relevant expertise from their organizations and colleagues in order to engage with CNCF discussions in a timely manner.
+## Emeritus
```

Comment:
Let's make it clearer that the Emeritus list is for past (elected) TOC Members (not Contributors). For similar reasons as above I'd recommend leaving them in a separate file